samples/sandbox/src/main/kotlin/com/bumble/appyx/sandbox/client/workflow/WorkflowExampleActivity.kt

```diff
@@ -75,21 +80,37 @@ class WorkflowExampleActivity : NodeActivity() {
         }
     }
 
-    private fun executeWorkflow1(): Job {
-        return lifecycleScope.launch {
+    private fun executeNavigation(navigationAction: suspend () -> Unit) {
+        job?.cancel()
+        job = lifecycleScope.launch {
+            navigationAction()
+        }
+    }
```

Comment:
I feel this should be part of `executeWorkflow` internally in `Node`. Not just to reduce code here, but the lifecycle scoping should belong to the individual nodes rather than the entire activity.

Reply:
> I feel this should be part of `executeWorkflow` internally in `Node`. Not just to reduce code here, but the lifecycle scoping should belong to the individual nodes rather than the entire activity.

This this the global scope. It will be cancelled if activity gets `onDestoy` callback. Individual nodes use their own `lifecycleScope` so they will be cancelled when the node is destoyed